src/classes/BDI_ObjectMappingLogic.cls

```diff
@@ -0,0 +1,46 @@
+public with sharing virtual class BDI_ObjectMappingLogic {
```

Comment:
Need the standard comment header on all class files

Reply:
Added